core/client/templates/-navbar.hbs

```diff
@@ -42,13 +42,31 @@
         {{#gh-dropdown tagName="div" classNames="dropdown" name="user-menu" closeOnClick="true"}}
             <ul class="dropdown-menu dropdown-triangle-top-right js-user-menu-dropdown-menu" role="menu">
                 <li role="presentation">{{#link-to "settings.users.user" session.user.slug classNames="dropdown-item user-menu-profile js-nav-item" role="menuitem" tabindex="-1"}}<i class="icon-user"></i> Your Profile{{/link-to}}</li>
-                <li role="presentation"><a class="dropdown-item user-menu-support" role="menuitem" tabindex="-1" href="http://support.ghost.org/"><i class="icon-support"></i> Help / Support</a></li>
                 <li class="divider"></li>
                 <li role="presentation">{{#link-to "signout" classNames="dropdown-item user-menu-signout" role="menuitem" tabindex="-1"}}<i class="icon-power"></i> Sign Out{{/link-to}}</li>
             </ul>
         {{/gh-dropdown}}
     </div>{{! .user-menu }}
 
+    <div class="nav-item help-menu">
+        {{#gh-dropdown-button dropdownName="help-menu" tagName="div" classNames="nav-label clearfix"}}
```

Comment:
I'm guessing you're doing `div` over `button` because of styling hardships; want to add a `role` for accessibility?